gb-cpu/src/processor/cpu/registers.rs

```diff
@@ -0,0 +1,132 @@
+pub mod area;
+
+use crate::bus::Bus;
+use area::{Bits16, Bits8};
+use std::fmt;
+
+#[derive(Debug, Default)]
+pub struct Registers {
+    a: u8,
+    f: u8,
+    b: u8,
+    c: u8,
+    d: u8,
+    e: u8,
+    h: u8,
+    l: u8,
+    sp: u16,
+    pub pc: u16,
+}
+
+impl fmt::Display for Registers {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
1. what's the difference between the `debug` and `display` format ?
2. Here we expecting to display the registers or to debug it ?

Reply:
There isn't any, that could be rewritten

Reply:
so do so :)